frontend/src/components/program/DialogActivityCreate.vue

```diff
@@ -16,21 +16,107 @@
       <slot name="activator" v-bind="scope" />
     </template>
 
-    <dialog-activity-form :activity="entityData" :period="period" />
+    <template #moreActions>
+      <copy-activity-info-dialog @closed="refreshCopyActivitySource">
+        <template #activator="{ on }">
+          <v-btn v-show="clipboardPermission === 'prompt'" v-on="on">
+            <v-icon left>mdi-information-outline</v-icon>
+            {{ $tc('components.program.dialogActivityCreate.copyPastActivity') }}
+          </v-btn>
+        </template>
+      </copy-activity-info-dialog>
+    </template>
+
+    <div v-if="hasCopyActivitySource">
+      <div class="mb-8">
+        <div v-if="!clipboardAccessDenied">
+          {{ $tc('components.program.dialogActivityCreate.clipboard') }}
+          <div style="float: right">
+            <small>
+              <a
+                href="#"
+                style="color: inherit; text-decoration: none"
+                @click="clearClipboard"
+              >
+                {{ $tc('components.program.dialogActivityCreate.clearClipboard') }}
+              </a>
+            </small>
+          </div>
+        </div>
+        <v-list-item
+          class="ec-copy-source rounded-xl blue-grey lighten-5 blue-grey--text text--darken-4 mt-1"
+        >
+          <v-list-item-avatar>
+            <v-icon color="blue-grey">mdi-clipboard-check-outline</v-icon>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title>
+              <CategoryChip
+                :category="copyActivitySource.category()"
+                class="mx-1"
+                dense
+              />
+              {{ copyActivitySource.title }}
+            </v-list-item-title>
+            <v-list-item-subtitle>
+              {{ copyActivitySource.camp().title }}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+          <v-list-item-action>
+            <e-checkbox
+              v-model="copyContent"
+              :label="$tc('components.program.dialogActivityCreate.copyActivityContent')"
```

Comment:
minor:
The label text content is a little long on mobile and hides the activity title.
But you see it in the title field and in the category field